app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ActiveRecord::Base
+  has_many :comments, foreign_key: :author_id
+  has_many :posts, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  private
+
+  def three_recent_post
+    posts.order('created_at Desc').limit(3)
+  end
```

Comment:
- [ ] I suggested making the method private which we are calling only from inside the class, not the `three_recent_post` which will be called from outside the class.  Remove the `private` from here.